### PR TITLE
update monitoring widget description

### PR DESCRIPTION
**Pull request by petrjasek**
picking fix from #1340

### CODE REVIEW

File: client/app/scripts/superdesk-desks/aggregate-widget/aggregate.js [hunk 1]
```diff
@@ -15,7 +15,8 @@ angular.module('superdesk.aggregate.widgets', ['superdesk.aggregate', 'superdesk
                 thumbnail: 'scripts/superdesk-desks/aggregate-widget/thumbnail.svg',
                 template: 'scripts/superdesk-desks/aggregate-widget/aggregate-widget.html',
                 configurationTemplate: 'scripts/superdesk-desks/aggregate-widget/configuration.html',
-                description: 'Displaying the monitoring content view.',
+                description: 'This widget allows you to create literally any content view you may need in Superdesk, be it production or ingest.' +
+                        'All you need is to select a desk, its stages or a saved search. Name your view once you are done. Enjoy!',
```
Comment (plamut) [inline client/app/scripts/superdesk-desks/aggregate-widget/aggregate.js]: Do we really want to have the `"Enjoy!"` bit in the enterprise software?
(personally I don't mind it at all, but someone else might :) )

**Comment by plamut** [comment]
:+1:

File: client/app/scripts/superdesk-desks/aggregate-widget/aggregate.js (hunk 1 above)
Comment (petrjasek) [inline client/app/scripts/superdesk-desks/aggregate-widget/aggregate.js]: it's by @tictactatic

Comment (tictactatic) [inline client/app/scripts/superdesk-desks/aggregate-widget/aggregate.js]: It's to entice people to edit the description further ;)

On Thu, Nov 5, 2015 at 10:46 AM, Petr Jašek <notifications@github.com>
wrote:

> In client/app/scripts/superdesk-desks/aggregate-widget/aggregate.js
> <https://github.com/superdesk/superdesk/pull/1345#discussion_r43992729>:
>
> > @@ -15,7 +15,8 @@ angular.module('superdesk.aggregate.widgets', ['superdesk.aggregate', 'superdesk
> >                  thumbnail: 'scripts/superdesk-desks/aggregate-widget/thumbnail.svg',
> >                  template: 'scripts/superdesk-desks/aggregate-widget/aggregate-widget.html',
> >                  configurationTemplate: 'scripts/superdesk-desks/aggregate-widget/configuration.html',
> > -                description: 'Displaying the monitoring content view.',
> > +                description: 'This widget allows you to create literally any content view you may need in Superdesk, be it production or ingest.' +
> > +                        'All you need is to select a desk, its stages or a saved search. Name your view once you are done. Enjoy!',
>
> it's by @tictactatic <https://github.com/tictactatic>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/superdesk/superdesk/pull/1345/files#r43992729>.
>



-- 
Sava Tatić
Managing director, Sourcefabric z.ú.
sava.tatic@sourcefabric.org
www.sourcefabric.org

Salvátorská 10
110 00 Praha 1, Czech Republic
+420 2 22 36 25 40
Skype: tictactatic | Twitter: @tictactatic | +1 647 889 2811 (Toronto)

Comment (plamut) [inline client/app/scripts/superdesk-desks/aggregate-widget/aggregate.js]: LOL ok :laughing:

**Comment by petrjasek** [comment]
e2e error is unrelated

**Pull request merged**